spec/lib/search_query_builder_spec.rb

```diff
@@ -172,6 +173,88 @@
     it "should not include an order query" do
       expect(query).not_to include("order")
     end
+
+    context "without stopwords" do
+      let(:params) {
+        {
+          "keywords" => "a mango"
+        }
+      }
+
+      it "should include stopwords in search" do
+        expect(query).to include("q" => "a mango")
+      end
+    end
+
+    context "with stopwords" do
+      let(:finder_content_item) {
+        {
+          'base_path' => '/find-eu-exit-guidance-business',
+          'details' => {
+            'facets' => facets,
+            'filter' => filter,
+            'reject' => reject,
+            'default_order' => default_order,
+            'default_documents_per_page' => 10
+          }
+        }
+      }
+
+      it "should not include stopwords in search" do
```

Comment:
Is it worth testing one of the contractions here too?  Like `shouldn't` or something else with an apostrophe

Reply:
@sihugh I've updated the test here: 7cbbf34a9bf4896f7aa2a6fdac29b8c9598c1919